src/ReactNavigation.re

```diff
@@ -0,0 +1,17 @@
+module Core = {
```

Comment:
nit: I think we don't need to keep the notion of "Core", "Native" and "Stack" inside this file. I would suggest we inline it to be:

"ReactNavigation.createAppContainer" as opposed to "ReactNavigation.Native.createAppContainer".

Reply:
The way I see it is that this is a interop file between React Navigation and BsReactNavigation, not a direct mappings per se.

Reply:
Sure, we should do it but currently, it's easier to know from which repository it's binding is coming.